src/main/java/camera/CameraChest.java

```diff
@@ -17,10 +17,12 @@
  * Class for describing a chest found in a camerastream/video/image.
  */
 public class CameraChest extends CameraObject {
-    private static final Scalar BOXCOLOUR_LOWER = new Scalar(18,100,100);
-    private static final Scalar BOXCOLOUR_UPPER = new Scalar(35,255,255);
+    private static final Scalar CHESTCOLOUR_LOWER = new Scalar(18, 100, 100);
+    private static final Scalar CHESTCOLOUR_UPPER = new Scalar(35, 255, 255);
+    private static final Scalar CHESTBOXCOLOUR = new Scalar(255, 0, 255);
```

Comment:
Shouldn't this be chestcolour in order to be consistent?

Reply:
Did checkstyle really give a warning/error about these names?

Reply:
no, because it is not the same thing